test/e2e/flask/user-operations.spec.ts

```diff
@@ -102,7 +102,16 @@ async function createSwap(driver: Driver) {
     swapFrom: 'TESTETH',
     swapTo: 'USDC',
   });
+
+  await driver.waitForSelector({ text: 'Swap', tag: 'button' });
```

Comment:
this won''t have any effect as the `clickElement` function already waits for the element to be visible and clickable